tiledb/bioimg/converters/base.py

```diff
@@ -164,7 +164,7 @@ def from_tiledb(
     @classmethod
     def to_tiledb(
         cls,
-        input_path: str,
+        input_path: Union[str, ImageReader],
```

Comment:
If this is not a `path` anymore let's rename it. At the top of my head just `input` would make sense.